spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkBatchQueryScan.java

```diff
@@ -46,7 +69,11 @@
   private final Long splitSize;
   private final Integer splitLookback;
   private final Long splitOpenFileCost;
+  private final List<Expression> runtimeFilterExpressions;
 
+  private TableScan scan = null; // lazy scan
```

Comment:
@rdblue, I've added a scan but I still build it in this class to reduce the amount of changes in this PR.